src/core/dbus-execute.c

```diff
@@ -952,6 +953,10 @@ static int property_get_exec_dir(
                 return r;
 
         FOREACH_ARRAY(i, d->items, d->n_items) {
+                /* Returned by property_get_exec_dir_symlink() to represent flags too */
+                if ((i->flags & _EXEC_DIRECTORY_PUBLIC) != 0)
```

Comment:
i don't get this comment?

it seems to me we should expose all dirs here, we have regardless if public flags are set or not?

Reply:
It was to avoid duplication, but ok